rollup.config.js

```diff
@@ -83,20 +113,25 @@ export default cliargs => [
         format: 'es',
         file: 'dist/video.es.js',
         strict: false,
-        banner
+        banner,
+        globals: globals.module
       }, {
         format: 'cjs',
         file: 'dist/video.cjs.js',
         strict: false,
-        banner
+        banner,
+        globals: globals.module
       }
     ],
+    external: externals.module,
     plugins: [
       alias({
         'video.js': path.resolve(__dirname, './src/js/video.js'),
         '@videojs/http-streaming': path.resolve(__dirname, './node_modules/@videojs/http-streaming/dist/videojs-http-streaming.es.js')
       }),
+      primedResolve,
```

Comment:
we specifically don't want to include resolve and cjs plugins for the es/cjs builds because then the dependencies are included in the build.

Reply:
Same with the core.js file below.

Reply:
If you run `npm run assets`, the esm and cjs builds should be significantly smaller than the main video.js file compare the output in this PR https://travis-ci.org/videojs/video.js/builds/445200591#L964-L968
```
┌─────────────────────────────┬───────────┬───────────┐
│ video.cjs.js                │   1.55 MB │ 369.78 KB │
├─────────────────────────────┼───────────┼───────────┤
│ video.es.js                 │   1.55 MB │ 369.73 KB │
├─────────────────────────────┼───────────┼───────────┤
│ video.js                    │    1.7 MB │ 392.46 KB │
└─────────────────────────────┴───────────┴───────────┘
```

vs master
```
┌─────────────────────────────┬───────────┬───────────┐
│ video.cjs.js                │   1.23 MB │ 290.51 KB │
├─────────────────────────────┼───────────┼───────────┤
│ video.es.js                 │   1.23 MB │ 290.46 KB │
├─────────────────────────────┼───────────┼───────────┤
│ video.js                    │   1.71 MB │ 394.92 KB │
└─────────────────────────────┴───────────┴───────────┘
```

Eventually, when we figure out how not to bundle VHS, it can be a smaller size difference.